RocketChat/Rocket.Chat
# [FIX] Can't delete file from Room's file list

## Proposed changes (including videos or screenshots)


### before
![image](https://user-images.githubusercontent.com/27704687/120215931-bb239700-c20c-11eb-9494-d4bc017df390.png)

### after
![image](https://user-images.githubusercontent.com/27704687/120216113-f8882480-c20c-11eb-9afb-b127e66a43da.png)



## Issue(s)


## Steps to test or reproduce

- Upload a new file
- Check if it's possible to delete it

## Further comments
